packages/base/field-component.gts

```diff
@@ -35,17 +36,30 @@ export function getBoxComponent(
     {},
     model,
     defaultFieldFormat(format),
-    actions
+    context
   );
 
   let component: ComponentLike<{ Args: {}; Blocks: {} }> = <template>
-    <Implementation
-      @model={{model.value}}
-      @fields={{internalFields}}
-      @set={{model.set}}
-      @fieldName={{model.name}}
-      @actions={{actions}}
-    />
+    {{! cardComponentModifier is used for the host environment to get access to card's rendered elements }}
+    {{#if context.cardComponentModifier}}
```

Comment:
Maybe we should have a default no-op modifier in order to avoid the if/then repetition?